test/integration/users.test.js

```diff
@@ -24,6 +24,14 @@ const {
 const { addJoinData, addOrUpdate } = require("../../models/users");
 const userStatusModel = require("../../models/userStatus");
 
+const userRoleUpdate = userData[4];
+const userRoleUnArchived = userData[9];
+const userAlreadyMember = userData[0];
+const userAlreadyNotMember = userData[9];
+const userAlreadyArchived = userData[5];
+const userAlreadyUnArchived = userData[4];
+const nonSuperUser = userData[0];
```

Comment:
Can we use separate variable names for different types of user data instead of indexes?

Reply:
Can we connect for this comment? Didn't understand :)

Reply:
sunny I did discussed with you...can you resolved this